account_banking_sepa_credit_transfer/tests/test_sct.py

```diff
@@ -153,7 +153,6 @@ def check_eur_currency_sct(self):
         self.assertEqual(agrolait_pay_line1.communication, 'F1341')
         self.payment_order.draft2open()
         self.assertEqual(self.payment_order.state, 'open')
-        self.assertEqual(self.payment_order.sepa, True)
```

Comment:
don't we want to replace this assertion by a check on the lines?

Reply:
Right. I added some checks in my commit https://github.com/OCA/bank-payment/pull/658/commits/dcd653f0bb578aa573ed6e6b5f7c924e2b3c4cf8